packages/mdc-textfield/_mixins.scss

```diff
@@ -486,24 +434,30 @@
   overflow: hidden;
 
   .mdc-text-field__input {
-    height: auto;
+    align-self: auto;
+    box-sizing: content-box;
+    height: 30px;
     margin: 0;
     padding: $padding-inset;
     padding-top: $padding-inset * 2;
     border: 1px solid transparent;
   }
 
   .mdc-floating-label {
+    @include mdc-rtl-reflexive-position(left, 1px);
     @include mdc-floating-label-fill-color($mdc-textarea-background);
-    @include mdc-rtl-reflexive(left, 1px, right, 0);
-    @include mdc-rtl-reflexive-property(margin, 8px, 0);
 
     top: $label-offset-y;
     bottom: auto;
+    width: auto;
```

Comment:
Use the `mdc-floating-label` width mixin

Reply:
Done!